code/modules/food_and_drink/plants.dm

```diff
@@ -1080,6 +1098,7 @@ ABSTRACT_TYPE(/obj/item/reagent_containers/food/snacks/plant)
 	slice_product = /obj/item/reagent_containers/food/snacks/plant/lemon/wedge
 	slice_amount = 6
 	slice_suffix = "wedge"
+	brew_result = "lemonade"
```

Comment:
One lemon = 20u of lemonade
One sliced lemon = 120u of lemonade?

Reply:
This is definitely more of a statement on outdated brewery code rather than an issue with the brew result. Will fix.

Reply:
How so?  As implemented that is the behavior, otherwise I would have possibly just expected the whole fruit to brew the result and then the children to clear the brew_result.